### PR TITLE
Use S3 assigned by Bucketeer

### PR DESCRIPTION
**Pull request by ayumin**
Using S3 accesses from assigned by Bucketeer Addon.

### CODE REVIEW

File: config/initializers/paperclip.rb [hunk 1]
```diff
@@ -21,9 +21,9 @@
   Paperclip::Attachment.default_options[:use_timestamp]  = false
 
   Paperclip::Attachment.default_options[:s3_credentials] = {
-    bucket: ENV.fetch('S3_BUCKET'),
-    access_key_id: ENV.fetch('AWS_ACCESS_KEY_ID'),
-    secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY'),
+    bucket: ENV.fetch('BUCKETEER_BUCKET_NAME'),
```
Comment (ykzts) [inline config/initializers/paperclip.rb]: What happens to people who used the environment variable "S3_BUCKET" so far?

Comment (beatrix-bitrot) [inline config/initializers/paperclip.rb]: agree with ykzts

maybe something like

bucket: ENV.fetch('S3_BUCKET') || ENV.fetch('BUCKETEER_BUCKET_NAME'),

would be better?

Comment (ayumin) [inline config/initializers/paperclip.rb]: Hi, I'm sorry. This PR is my mistake. But I willing to work more better for this PR, if some one wants to run Mastodon on Heroku well.
Thanks.

Comment (beatrix-bitrot) [inline config/initializers/paperclip.rb]: @ayumin will you be updating this, or should this PR be closed?

**Pull request closed without merging**